app/server.py

```diff
@@ -101,8 +103,23 @@ async def run_workflow(
 
     async with httpx.AsyncClient(verify=False, timeout=60.0) as client:
         for operation in workflow:
+            operation_services = []
+            runner_parameters = operation.pop("runner_parameters", {})
+            if "allowlist" in runner_parameters.keys():
+                for service in SERVICES[operation["id"]]:
+                    if service["id"] in runner_parameters["allowlist"]:
+                        operation_services.append(service)
+            else:
+                for service in SERVICES[operation["id"]]:
+                    operation_services.append(service)
+                if "denylist" in runner_parameters.keys():
+                    for service in operation_services:
+                        if service["id"] in runner_parameters["denylist"]:
+                            operation_services.remove(service)
+            logger.debug(f"Service providers to query for operation '{operation}':'{operation_services}'")
+
             service_operation_responses = []
-            for service in SERVICES[operation["id"]]:
+            for service in operation_services:
```

Comment:
We should have a check prior to this line that we actually have any operation_services. If we don't we should send something helpful. See #36